### ISSUE TITLE
Lock screen: circle message cannot be changed (easily) any more

### ISSUE THREAD

**Issue by ruedigerkupper** [ubports/ubuntu-touch#1740]
- Device: Xperia X (suzu)
- Channel: devel
- Build: 2021-05-26

### Steps to reproduce
Try double-tapping on the lock screen to change the message displayed in the notification circle

### Expected behavior
The message changes with each double-tap

### Actual behavior
The message does not change unless you keep tapping frantically at the display

### Logfiles and additional information
Bug seems to have appeared already some time ago. I could not check if it affects all devices or is specific to the Xperia X. Please confirm.

**Comment by lutin11** [comment]
Same on Vola phone

**Comment by Flohack74** [comment]
@UniversalSuperBox  that seems to happen since the first tap pulls the launcher from the left, that might eat up the double-tap event. Single tap however just briefly shows the launcher. nothing with press-and-hold or double tap will actually open it fully, so I think that can be called a regression from our various adventures with the app drawer and launcher...

**Comment by UniversalSuperBox** [comment]
@Flohack74, no, this is due to Qt changing how double-taps are handled in Qt. QtMir needs an implementation of https://github.com/ubports/qtubuntu/pull/20

**Comment by Flohack74** [comment]
Oh alright!

**Comment by ruedigerkupper** [comment]
I can cinfirm that "single-tap double-tap" seems to be the best working combo at the moment.

**Comment by kugiigi** [comment]
Yeah, triple tap seems to be the workaround for now.
